functions/zram.bats

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bats 
```

Comment:
As this function will not be part of all standard installations I would say this is classed as a `destructive` test. 
See: https://github.com/openhab/openhabian/blob/master/CONTRIBUTING.md#destructive-verification-tests-destruct-name

```
#!/usr/bin/env bats 
load zram

@test "destructive-zram" {
  run init_zram_mounts install
  [ "$status" -eq 0 ]

 ### Insert testcode from check_zram_mounts() ###
This make sure that we seperate test-code from production-code.
}
```

Reply:
@mstormi missed this